FRINXio/frinx-frontend
# fix code editor

| Q                        | A 
| ------------------------ | ---
| Fixed Issue?             | `FD-565` 
| Tests Added + Pass?      | Yes
| Any Dependency Changes?  |

https://frinxhelpdesk.atlassian.net/jira/software/c/projects/FD/boards/77?modal=detail&selectedIssue=FD-565
added content type selector when editing a http task to switch between editor languages

HOW TO TEST
Edit any workflow and add a HTTP task, switch to input parameters, select method to PUT (for instance) and play around with content type selector, editor should switch language based on selected content type.
